### DISCUSSION TITLE
hacktoberfest 2021 Как результаты?

### DISCUSSION THREAD

**Discussion by tst32** [doka-guide/content#2243] [Разное]
https://hacktoberfest.digitalocean.com
Интересно узнать

**Comment by nlopin** [comment]
У нас нет результатов. Там даже нет какой-то отдельной странички для мейтейнеров, но по нашей статистике за октябрь мы вмёржили 75 пул-реквестов от 33 человек, в среднем больше двух реквестов на человека
